src/lib/prover/prover.ml

```diff
@@ -56,6 +56,9 @@ module Worker_state = struct
     val toggle_internal_tracing : bool -> unit
 
     val set_itn_logger_data : daemon_port:int -> unit
+
+    val get_blockchain_verification_key :
```

Comment:
I wonder also what to do in Check | No_check modes. Where we return dummy keys. Maybe this method should return `Pickles.Verification_key.t Deferred.t` but then in Mina_grahpql modules we are exposing Verifiaction key from verifier so we would need to either break the interface and also there wrap repsone in optional or handle returning diummy as well

Reply:
Let's update graphql module to use key that is returned from prover. It's okay if it's dummy in case of non-full modes.

It's non-ideal that we create a dummy prover that returns dummy keys. I'd prefer it to be refactored to not initialize prover unless it's full mode. But let's save this idea for one of next PRs, not this one.